### PR TITLE
Implement display profile UI 

### LINKED ISSUE #25: UI task: Presenting Output
Present algorithm output to user
From #5

### PR DESCRIPTION
**Pull request by jenny-l-li**
- Add UI to display profile
- Cleaned up old UI (navigation bar, results page)
- Results page now displays 4 personality outputs with collapsible dropdowns
closes #25

**TODO:** 
- Replace filler text with actual descriptions of personality outputs

https://user-images.githubusercontent.com/32944104/108580379-1d046480-72e0-11eb-8020-6c51babf1a50.mov

### CODE REVIEW

File: flaskr/__init__.py [hunk 1]
```diff
@@ -94,7 +94,7 @@ def results():
 		else:
 			return redirect(url_for('index'))
 		if 'code' in session:
```
Comment (elrodrigues) [inline flaskr/__init__.py:96]: I want to add @khzheng24998's commit from #34 here after resolving merge conflicts

**Review by elrodrigues — CHANGES_REQUESTED** [review]
The changes are solid.

The major issue is resolving conflicts with `master`, @mjessicacheng's #34 and this pull request. I want to add @khzheng24998's bug fix to this too, and possibly implement the TODO, so I'll experiment with resolving merge conflicts locally. If I have something that works, I'll recommend way to merge #33 and #34 into `master`.

**Comment by khzheng24998** [comment]
@jenny-l-li Since Jessica undid (is that really the past tense form of undo?) most of her changes from #34, I think that it should be safe for you to merge these changes into master now. The other changes can be added back into the PR for #34 after your changes are merged in.

**Review by elrodrigues — APPROVED** [review]
Changes are solid.

I plan to merge `ui-profile` to master and then merge the updated master to `compare-endpoint` so Jessica can add her changes.

**Pull request merged**